style.css

```diff
@@ -1,3 +1,7 @@
 body{
```

Comment:
I kindly suggest that you apply correct spacing in your CSS to make your code easy to read and go through